pkg/client/daemon/dns/flush_windows.go

```diff
@@ -0,0 +1,9 @@
+package dns
+
+import (
+	"context"
+)
+
+func Flush(c context.Context) {
+	// TODO
```

Comment:
I don't think `setDNS()` performs a complete flush. It will just flush the device. Windows may have other caches in other places and we should find out where and implement a proper flush here. I'll update the TODO comment.

Reply:
Doh. You're right. The `setDNS()` calls `ipconfig /flushdns`.